agcaballero/portfolio/src/main/webapp/style.css

```diff
@@ -88,6 +96,75 @@ a:visited {
   background-color: #3d405b; /* independence */
 }
 
+/* Welcome Page */
+#welcome-div {
+  border-radius: 20px;
+  height: 125px;
+  margin: 75px;
+}
+
+#welcome-sign {
+  font-size: 150px;
+  margin: 0px;
+}
+
+#squares {
+  color: white;
+  flex-direction: row;
+  font-family: Georgia, serif;
+}
+
+#squares a,
+#squares a:visited {
+  color: white;
+  text-decoration: none;
+}
+
+#squares h2 {
+  font-size: 30px;
+  margin-bottom: 5px;
+}
+
+#squares h3 {
+  color: white;
+  margin-top: 5px;
+}
+
+#squares hr {
+  border-top: 2px solid;
+  width: 80%;
+}
+
+.square {
+  align-content: center;
+  background-color: #e07a5f;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  height: 300px;
+  justify-content: center;
+  margin-top: 0px;
+  margin: 30px;
```

Comment:
Does `margin` conflict with `margin-top`, since `margin: 30px` means 30px on all sides?

Reply:
I think margin-top overrides it. However, I can just set a more custom margin for this element to make it less confusing.

Reply:
Done